### PR TITLE
Fix week browser breaking as year changes

### PR DESCRIPTION
**Pull request by joonashak**
Week browsing was implemented based on a week parameter in the path that consisted of just a week number or a year and a week number. This proved to be insufficient to handle the last and first weeks of year.

- Week browsing is now based on an unambiguous date.
- An error message is shown if the user navigates to a path containing an invalid week param.
- Also, Playwright updated due to the old version not being installable anymore ⚰️ 

https://funidata.atlassian.net/browse/KEIJO-132

### CODE REVIEW

File: web/src/i18n/fi.ts [hunk 1]
```diff
@@ -37,6 +37,9 @@ const fi = {
       "Palkansaajanumeroasi ei löytynyt HTTP-otsakkeista. Keijoa ei voi käyttää ilman tätä tietoa. Ilmoita ongelmasta esihenkilöllesi.",
     unknownError: "Keijo kohtasi tuntemattoman virheen. Ota yhteyttä Kettu-tiimiin.",
     notFound: "Pyydettyä sivua ei löytynyt. Tarkasta osoite.",
+    oldWeekParamFormat:
+      "Sivun osoitteessa on vanhan muotoinen viikkoparametri. Mene ensin etusivulle, jonka jälkeen voit jatkaa käyttöä normaalisti.",
+    invalidWeekParam: "Anettu viikkoparametri ei ole kelvollinen.",
```
Comment (rrimpila) [inline web/src/i18n/fi.ts:42]: Typo: 'Annettu'

Comment (joonashak) [reply web/src/i18n/fi.ts:42]: 🆘 

Screenshot 2024-12-17 at 8 37 35

Comment (joonashak) [reply web/src/i18n/fi.ts]: Fixed 🚀

**Review by rrimpila — APPROVED** [review]
LG2M, I can now browse weeks to next year and even back!

**Pull request merged**